arviz/plots/kdeplot.py

```diff
@@ -147,11 +149,15 @@ def plot_kde(
         rug_kwargs.setdefault("marker", "_" if rotated else "|")
         rug_kwargs.setdefault("linestyle", "None")
         rug_kwargs.setdefault("color", default_color)
+        rug_kwargs.setdefault("space", 0.2)
 
         plot_kwargs.setdefault("linewidth", linewidth)
         rug_kwargs.setdefault("markersize", 2 * markersize)
 
         density, lower, upper = _fast_kde(values, cumulative, bw)
+
+        rug_space = max(density) * rug_kwargs.get("space")
```

Comment:
@aloctavodia You could use a dictionary pop which will remove the keyword and value from the dictionary, saving you from having to manually delete later

`rug_space = max(density) * rug_kwargs.pop("space")`

Admittedly I haven't run the code yet, just what I surmise from static reading.

Reply:
pop will KeyError if not found. pop witj default will not

    dict.pop('a', '1')

Reply:
Thanks for the suggestions! Using `rug_kwargs.pop("space")` will always work in this scenario because space is set a few line above.